caasp-master.yaml

```diff
@@ -0,0 +1,116 @@
+heat_template_version: 2017-09-01
```

Comment:
It will work also with 2016-10-14

Reply:
Thx! Changed it in the main caasp-multi-master-stack.yml but missed it here in included caasp-master.yml